proxyd/server.go

```diff
@@ -411,6 +458,10 @@ func (s *Server) handleBatchRPC(ctx context.Context, reqs []json.RawMessage, isL
 			continue
 		}
 
+		if (parsedReq.Method == "eth_newFilter" || parsedReq.Method == "eth_getLogs") && !s.allowedBlockRange(parsedReq) {
+			responses[i] = NewRPCErrorRes(parsedReq.ID, errors.New("block range too large"))
+		}
```

Comment:
Looks like this is missing a `continue` here unless I'm misunderstanding
```suggestion
		    continue
		}
```

Reply:
good catch! thanks